src/test/regress/expected/grant_on_database_propagation.out

```diff
@@ -0,0 +1,967 @@
+--below test should fail and should throw an error since myuser_1 still have the dependent privileges
+revoke grant option for CREATE,CONNECT,TEMP,TEMPORARY on database regression from myuser restrict ;
+ERROR:  dependent privileges exist
+HINT:  Use CASCADE to revoke them too.
+--below test should succeed and should not throw any error since myuser_1 privileges are revoked with cascade
+revoke grant option for CREATE,CONNECT,TEMP,TEMPORARY on database regression from myuser cascade ;
+--here we test if myuser still have the privileges after revoke grant option for
+select has_database_privilege('myuser','regression', 'CREATE');
+ has_database_privilege
+---------------------------------------------------------------------
+ t
+(1 row)
+
+select has_database_privilege('myuser','regression', 'CONNECT');
+ has_database_privilege
+---------------------------------------------------------------------
+ t
+(1 row)
+
+select has_database_privilege('myuser','regression', 'TEMP');
+ has_database_privilege
+---------------------------------------------------------------------
+ t
+(1 row)
+
+select has_database_privilege('myuser','regression', 'TEMPORARY');
+ has_database_privilege
+---------------------------------------------------------------------
+ t
+(1 row)
+
+\c - - - :worker_1_port
+select has_database_privilege('myuser','regression', 'CREATE');
+ has_database_privilege
+---------------------------------------------------------------------
+ t
+(1 row)
+
+select has_database_privilege('myuser','regression', 'CONNECT');
+ has_database_privilege
+---------------------------------------------------------------------
+ t
+(1 row)
+
+select has_database_privilege('myuser','regression', 'TEMP');
+ has_database_privilege
+---------------------------------------------------------------------
+ t
+(1 row)
+
+select has_database_privilege('myuser','regression', 'TEMPORARY');
+ has_database_privilege
+---------------------------------------------------------------------
+ t
+(1 row)
+
+\c - - - :master_port
+reset role;
+revoke  CREATE,CONNECT,TEMP,TEMPORARY on database regression from myuser;
+revoke CREATE,CONNECT,TEMP,TEMPORARY on database regression from myuser_1;
+drop user myuser_1;
+drop user myuser;
+---------------------------------------------------------------------
+-- test CREATE,CONNECT,TEMP,TEMPORARY privileges one by one on database multi database
+-- and multi user
+create user myuser;
+create user myuser_1;
+create database test_db;
+NOTICE:  Citus partially supports CREATE DATABASE for distributed databases
+DETAIL:  Citus does not propagate CREATE DATABASE command to workers
+HINT:  You can manually create a database and its extensions on workers.
+SELECT result FROM run_command_on_workers($$create database test_db$$);
+     result
+---------------------------------------------------------------------
+ CREATE DATABASE
+ CREATE DATABASE
+(2 rows)
+
+revoke connect,temp,temporary  on database test_db from public;
```

Comment:
nit: this line seems unnecessary since it's already done at the start of the file.